### PR TITLE
CDPT-2150 - Update logo error pages.

### PR DESCRIPTION
**Pull request by EarthlingDavey**

### CODE REVIEW

**Comment by EmilyHazlehurst** [comment]
@EarthlingDavey looks good to me!

Just a thought regarding the alt text on the logo in the header and the footer; do we need it? I'm not sure there's much value in "MoJ logo" (especially as it's read as "modge logo 😅 ") or "Crown copyright" as descriptions. Crown copyright is repeated immediately after and the header link describes it's function.

The examples on GDS don't have alt text for the crown logos. The footer version is just a link with a background image and the header version is an svg that's not focusable so only the surrounding link context is read out. 

I'll make an enhancement ticket :)

**Review by EmilyHazlehurst — APPROVED** [review]

File: web/app/themes/clarity/error-pages/maintenance.html [hunk 1]
```diff
@@ -70,7 +70,7 @@ <h1 class="o-title o-title--page">Website temporarily unavailable</h1>
     <div class="l-primary"></div>
     <div class="l-secondary c-copyright">
       <div class="c-copyright-notice">
-        <img src="/app/themes/clarity/dist/images/crown_copyright_logo.png" alt="crown copyright logo">
+        <img src="/app/themes/clarity/dist/images/tudor_crest.svg" alt="crown copyright logo">
```
Comment (EmilyHazlehurst) [inline web/app/themes/clarity/error-pages/maintenance.html:73]: Have these logos been added in a previous commit? tudor_crest and moj_logo_header?

Comment (EarthlingDavey) [reply web/app/themes/clarity/error-pages/maintenance.html:73]: Good spot, yes they were :)

**Comment by EarthlingDavey** [comment]
@EmilyHazlehurst  Thanks for the review and explaining about the alt text. I agree, let's get this through and pick up the accessibility improvement at a later date :)

**Pull request merged**